.github/workflows/ci-nix.yml

```diff
@@ -332,3 +336,39 @@ jobs:
         webhook: ${{ secrets.DISCORD_WEBHOOK }}
         # current job is a success, but that's not what we're interested in
         status: failure
+
+  deb-binary:
+    name: "Bundling for DEB and RPM"
+    runs-on: ubuntu-20.04 
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v3
+        if: github.ref == 'refs/heads/master'
```

Comment:
The whole job should be conditional. Making single steps conditional makes later steps fail right now and adding the same if condition to all of them to fix that seems suboptimal.